### DISCUSSION TITLE
Cleaning up context window

### DISCUSSION THREAD

**Discussion by ghost** [yetone/avante.nvim#1015] [Q&A]
Am I missing something, is there a way to be able to clear the context from the chat?

Say I want to solve problem A using the chat. I give it the code, apply the code then move on.

Then later in that same file if I want to solve problem B, can I clear the previous inputs to avoid flooding the context window? 
So I am not sending 1000s of tokens, running up the API credits used?

**Comment by Rosbifbr** [comment]
Yeah, this is a much needed feature for me too. It's either not very well documented or inexisting.

**Comment by tepavcevic** [comment]
There are slash commands `/clear` and `/reset` available for that. You can get a list of commands via executing `/help` in avante chat.

**Reply by ibehnam** [reply]
it'd be nice to assign these to shortcut keys, like <leader>ar for reset and <leader>ac for clean.

**Reply by keyvhinng** [reply]
@tepavcevic what is the difference between `/clear` and `/reset` ?
